test/integration/resultsPage.js

```diff
@@ -40,10 +40,48 @@ describe('Results page', () => {
 describe('Results page error handling', () => {
   describe('when search is not included', () => {
     it('should return a descriptive error messages', (done) => {
+      const search = null;
+      const errorMessage = messages.emptySearchMessage();
```

Comment:
Maybe for tests we should be more explicit than implicit. This does mean the tests will fail when we make content changes, which forces us to change the tests. If we accept that this is how we roll, I'm happy to change it.